### PR TITLE
Add support for detecting blockpages in India

### PR DESCRIPTION
**Pull request by hellais**

### CODE REVIEW

File: pipeline/batch/sql_tasks.py [hunk 1]
```diff
@@ -147,6 +148,24 @@ def identified_vendors(metrics_table):
                 LIKE '%squid%'
             OR {metrics_table}.test_keys -> 'received' ->> 3
                 LIKE '%squid%')
+UNION
```
Comment (anadahz) [inline pipeline/batch/sql_tasks.py:151]: Is this addition really related to the identification of IN block pages?

Comment (hellais) [inline pipeline/batch/sql_tasks.py:151]: Actually no, this is related to privoxy detection. I guess the commit message is indeed a bit misleading.
